src/node_process_events.cc

```diff
@@ -21,7 +21,12 @@ using v8::Value;
 Maybe<bool> ProcessEmitWarningSync(Environment* env, std::string_view message) {
   Isolate* isolate = env->isolate();
   Local<Context> context = env->context();
-  Local<String> message_string = OneByteString(isolate, message);
+  v8::Local<v8::String> message_string =
+      v8::String::NewFromUtf8(isolate,
+                              message.data(),
+                              v8::NewStringType::kNormal,
+                              static_cast<int>(message.size()))
+          .ToLocalChecked();
```

Comment:
```suggestion
  Local<String> message_string;
  if (!String::NewFromUtf8(isolate,
                              message.data(),
                              NewStringType::kNormal,
                              static_cast<int>(message.size()))
          .ToLocal(&message_string)) {
    return Nothing<bool>();
  }
```

Reply:
yes a error management here is much healthier, thanks